Apps/WebClient/src/ClientApp/src/services/restEncounterService.ts

```diff
@@ -67,21 +68,26 @@ export class RestEncounterService implements IEncounterService {
 
     public getHospitalVisits(
         hdid: string
-    ): Promise<RequestResult<HospitalVisit[]>> {
+    ): Promise<RequestResult<HospitalVisitResult>> {
         return new Promise((resolve, reject) => {
             if (!this.isEnabled) {
```

Comment:
You'll want to update this to be dependent on whether the "HospitalVisit" module is enabled instead of the "Encounter" module.

`isEnabled` currently reflects whether the "Encounter" module is enabled.  I'd suggest renaming it to `isEncounterEnabled` and adding a new `isHospitalVisitEnabled`.